docs/sqlalchemy.txt

```diff
@@ -399,3 +400,63 @@ delete a single table use ``drop(...)`` as shown below::
 
     >>> engine.dialect.has_table(conn, table_name='departments')
     False
+
+
+Insert From Select
+==================
+
+In SQLAlchemy, the ``insert().from_select()`` function returns a new ``Insert``
+construct which represents an `INSERT...FROM SELECT` statement. 
+This functionality is now supported by the `crate` client library.  
+Here is an example that uses ``insert().from_select()``::
+
+
+First let's define and create the tables::
+
+    >>> from sqlalchemy import select, insert
+
+    >>> class Todos(Base):
+    ...     __tablename__ = 'todos'
+    ...     __table_args__ = {
+    ...         'crate_number_of_replicas': '0'
+    ...     }
+    ...     id = sa.Column(sa.String, primary_key=True, default=gen_key)
+    ...     content = sa.Column(sa.String)
+    ...     status = sa.Column(sa.String)
+
+    >>> class ArchivedTasks(Base):
+    ...     __tablename__ = 'archived_tasks'
+    ...     __table_args__ = {
+    ...         'crate_number_of_replicas': '0'
+    ...     }
+    ...     id = sa.Column(sa.String, primary_key=True)
+    ...     content = sa.Column(sa.String)
+
+    >>> Base.metadata.create_all(bind=engine)
+
+Let's add a task to the Todo table::
+
+    >>> task = Todos(content='Write Tests', status='done')
+    >>> session.add(task)
+    >>> session.commit()
+
+    >>> from time import sleep
+    >>> sleep(2)
+
+Using ``insert().from_select()`` to archive the task in ArchivedTasks table::
+
+    >>> sel = select([Todos.id, Todos.content]).where(Todos.status=="done")
+    >>> ins = insert(ArchivedTasks).from_select(['id','content'], sel)
+    >>> result = session.execute(ins)
+    >>> session.commit()
+
+    >>> from time import sleep
+    >>> sleep(2)
```

Comment:
please use refresh statements instead of sleep. Sleep will just cause this stuff to be flaky in some form.